src/pcapi/sandboxes/scripts/getters/pro_10_desk.py

```diff
@@ -19,3 +22,23 @@ def get_existing_pro_validated_user_with_validated_offerer_with_validated_user_o
         "booking": get_booking_helper(booking),
         "user": get_pro_helper(user_offerer.user),
     }
+
+
+def get_existing_pro_validated_user_with_validated_offerer_with_validated_user_offerer_with_eac_offer_with_stock_with_not_used_booking_USED_BY_INSTITUTE():
```

Comment:
```suggestion
def get_existing_pro_validated_user_with_validated_offerer_with_validated_user_offerer_with_eac_offer_with_stock_with_not_used_by_institute_booking():
```

Le mettre en caps lock c'est vraiment un détail d'implémentation là :sweat_smile: